apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -841,7 +842,24 @@ const EventTypePageWrapper: React.FC<PageProps> & {
   const { data } = trpc.viewer.eventTypes.get.useQuery({ id: props.type });
 
   if (!data) return null;
-  return <EventTypePage {...(data as EventTypeSetupProps)} />;
+
+  const eventType = data.eventType;
+  const eventTypeWorkflows = eventType.workflows.map((workflowRel) => workflowRel.workflow);
+
+  const { data: workflows, isPending } = trpc.viewer.workflows.getAllActiveWorkflows.useQuery({
+    eventTypeWorkflows: eventTypeWorkflows,
+    teamId: eventType.team?.id,
+    userId: eventType.owner?.id,
+  });
+
+  if (isPending) return null;
```

Comment:
Can be improved in a follow-up but I think we shouldn't block the rendering of entire event-type page just for the workflows count.

Reply:
Looks like this wasn't needed and actually introduced the [toggle status bug ](https://github.com/calcom/cal.com/pull/15083#issuecomment-2165019863)